src/content/configuration/other-options.md

```diff
@@ -60,29 +60,167 @@ This will force webpack to exit its bundling process.
 
 `boolean` `object`
 
-Cache the generated webpack modules and chunks to improve build speed. Caching is enabled by default while in watch mode. To disable caching simply pass:
+Cache the generated webpack modules and chunks to improve build speed. `cache` is set to `type: 'memory'` in [`development` mode](/concepts/mode/#mode-development) and disabled in [`production` mode](/concepts/mode/#mode-production). To disable caching simply pass `false`:
```

Comment:
Add a note that `cache: true` is an alias to `cache: { type: "memory" }`

Reply:
Remove `simply` that makes the reader feel stupid.

Reply:
added a note and removed "simply" word